dags/shredder.py

```diff
@@ -21,8 +21,11 @@
 base_command = [
     "script/shredder_delete",
     "--state-table=moz-fx-data-shredder.shredder_state.shredder_state",
-    "--start-date={{'2019-11-20' if ds == '2020-03-09' else macros.ds_add(ds, -7*4*2)}}",
-    "--end-date={{ds}}",
+    # start date should be two schedule intervals before end date, 27-28*2=-29
+    "--start-date={{macros.ds_add(ds, -29)}}",
+    # dags run schedule_interval after ds, and end date should be one day
+    # before the dag runs, so 27 days after ds
+    "--end-date={{macros.ds_add(ds, 27)}}",
```

Comment:
I see, this is because of how Airflow sets shifts the date a whole interval from the run date.

Reply:
yep.